ci/ci/build.py

```diff
@@ -13,6 +13,13 @@
 log = logging.getLogger('ci')
 
 
+log_pretty_print = "jq -Rrc '. as $raw | try \
```

Comment:
Nice, your jq-fu is strong.  I don't think `-c` (print json compact) is necessary with `-r` (raw output).  Nit: prefer natural English: pretty_print_long.